Src/GoogleApis.Auth.DotNet4/GoogleApis.Auth.DotNet4.csproj

```diff
@@ -33,6 +33,9 @@
     <DocumentationFile>Google.Apis.Auth.PlatformServices.xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="BouncyCastle.Crypto">

```

Comment:
You will have to change the following file as well:
https://github.com/google/google-api-dotnet-client/blob/master/Tools/Google.Apis.Release/Resources/Google.Apis.Auth.VERSION.nuspec
It's part of our release process of the NuGet packages.
I think you should do something like, what we are doing here:
https://github.com/google/google-api-dotnet-client/blob/master/Tools/Google.Apis.Release/Resources/Google.Apis.VERSION.nuspec

<dependencies>
        <group>
          <dependency id="Google.Apis.Core" version="VERSION" />
        </group>
        <group targetFramework=".NETFramework4.0">
          <dependency id="Google.Apis.Core" version="VERSION" />
          <dependency id="<BOUNCY HERE>" version="<BOUNCY-VERSION-HERE" />
        </group>
      </dependencies>